cli/worker.rs

```diff
@@ -415,7 +414,7 @@ impl CliMainWorker {
 
 #[derive(Clone)]
 pub struct CliMainWorkerFactory {
-  shared: Arc<SharedWorkerState>,
+  pub shared: Arc<SharedWorkerState>,
```

Comment:
Are these visibility changes needed?

Reply:
No not at all, I used that while inspecting, I'll remove it soon.